.travis.yml

```diff
@@ -0,0 +1,34 @@
+sudo: required
+
+language: python
+
+services:
+  - docker
+
+env:
+  global:
+    - PORTS="8069:8069"
+    - LINKS="db:db"
+    - DAEMONIZE="1"
+    - ODOO_VERSION="10.0"
+
+  matrix:
+    - TESTS="1" ODOO_VERSION="10.0" DOCKER_CONTEXT="./tests/odoo10"
+    - TESTS="1" ODOO_VERSION="9.0" DOCKER_CONTEXT="./tests/odoo9"
+    - LINT_CHECK="1"
+
+before_install:
+  - docker build -t "tecnativa/odoo-base:${ODOO_VERSION}" --build-arg "ODOO_VERSION=${ODOO_VERSION}" ./
+  - docker pull kiasaki/alpine-postgres
+  - docker run -d -it -e POSTGRES_USER="odoo" -e POSTGRES_PASSWORD="odoopassword" --name="db" postgres:9.6-alpine
```

Comment:
This would go back into changing our build tools, which TBH I don't really have time for at the moment. I feel that it would be best to merge as is, then polish things like this as we go forward. I agree this isn't the best design, but it works and it's better than what you have now.